docs/manual/nullness-checker.tex

```diff
@@ -97,6 +97,15 @@
   See Section~\ref{nullness-arrays} for a discussion.
   % TODO: this option is temporary and the sound option
   % should become the default.
+
+\item
+  \label{nullness-lint-trustarraylenzero}%
+  If you supply the \code{-Alint=trustArrayLenZero} command-line option, then a
+  call to \code{Collection.toArray} will return an array with a non-null
```

Comment:
"will return" implies a run-time behavior.  I think you mean that the type-checker will treat the type of an expression to be non-null.

Also, this sentence is long and hard to follow.  Maybe shorten it to "the checker will trust `@ArrayLen(0)` annotations" and the cross-reference.